src/com/dakl/Main.java

```diff
@@ -362,6 +362,7 @@ Space getLocation(int piece_id) {
         return getLocation(found);
     }
 
+    // REVIEW : method naming
```

Comment:
It is often beneficial to be more verbose in method naming and be very clear about what the method does -e .g. maybe this should have been getAllPieces(). 

I was also surprised to see that a seemingly "getter" method (that seems should just be retrieving stuff) goes and modifies the internal state.